### PR TITLE
#152919323 Relate log and session

### PR DESCRIPTION
**Pull request by CeciliaCaroline**
#### What does this PR do?
Set the work out log id as a foreign key in the workout sessions table.

#### Description of Task to be completed?
Create a foreign key relationship between the workout log and session

#### How should this be manually tested?
Query the workout log table in the database with the following commands`select * from manager_workoutlog;` 

#### Any background context you want to provide?
Right now the workout logs and the workout session are only linked through the date (for sessions this value is unique and enforced by the DB). This presents in some places a problem, since it's not possible to just follow a foreign key relationship with the ORM like in other places.

#### What are the relevant pivotal tracker stories?
[#152919323](https://www.pivotaltracker.com/story/show/152919323)

### CODE REVIEW

**Review by VincentHokie — APPROVED** [review]

**Review by Sharonsyra — COMMENTED** [review]
Nice work

**Comment by Sharonsyra** [comment]
@CeciliaCaroline why are you merging to master?

**Comment by Sharonsyra** [comment]
@CeciliaCaroline why the many files changed?

File: wger/manager/models.py [hunk 1]
```diff
@@ -42,7 +42,6 @@
 )
 from wger.utils.fields import Html5DateField
 
-
 logger = logging.getLogger(__name__)
```
Comment (jonathankamau) [inline wger/manager/models.py:47]: Why this space?

**Review by jonathankamau — CHANGES_REQUESTED** [review]
Your TravisCI build is failing. Please fix that.

**Review by jonathankamau — CHANGES_REQUESTED** [review]
This PR is almost ready for merging. change the PR title to `#152919323 Relate log and session` You don't  need to put `Ft` in the title. Also the PT ID in the PR description should be a link.

**Comment by CeciliaCaroline** [comment]
PR title changed to #152919323 Relate log and session and PT ID changed to link

**Review by jonathankamau — CHANGES_REQUESTED** [review]
Please remove the migration files. They are not necessary anymore.

**Comment by CeciliaCaroline** [comment]
@jonathankamau I removed the migrations in the last commit i made it was one of the causes of the merge conflict

**Pull request merged**